addon/components/bsy-input-addon/input.js

```diff
@@ -4,11 +4,12 @@ import layout from '../../templates/components/bsy-input-addon/input';
 const BsyInputAddonInputComponent = Component.extend({
   layout,
   tagName: "div",
+  classNameBindings: ["value:filled", "placeholder:filled"],
```

Comment:
this still doesn't work for input's type="data" :/

Reply:
as the placeholder is not control by html dom

Reply:
Works for me. The only thing that's not working is when you focus the field and then unfocus it. That's when `filled` class is lost.